### ISSUE TITLE
add to travis.yml running with ruby 2.3 as well

### ISSUE THREAD

**Issue by abonas** [hawkular/hawkular-client-ruby#51]

**Comment by jkremser** [comment]
I'd wait with this issue till it is supported by travis. Or the version of rvm travis uses.

``` bash
$ rvm use 2.3 --install --binary --fuzzy
ruby-2.3 is not installed - installing.
Searching for binary rubies, this might take some time.
Requested binary installation but no rubies are available to download, consider skipping --binary flag.
Gemset '' does not exist, 'rvm ruby-2.3 do rvm gemset create ' first, or append '--create'.
The command "rvm use 2.3 --install --binary --fuzzy" failed and exited with 2 during .
```

source: https://travis-ci.org/Jiri-Kremser/simple-websocket-vcr/jobs/124789533#L119

**Comment by abonas** [comment]
imo it's better to add it as 2.3.0 (that works) to be compliant with miq who test themselves with 2.3.0 as well. the travis issue is known and reported, pls see here https://github.com/abonas/kubeclient/pull/148

**Comment by jkremser** [comment]
Right, you are correct, I just find it out here http://rubies.travis-ci.org for Ubuntu 12.04 the 2.3.0 should be there

**Comment by abonas** [comment]
> Right, you are correct, I just find it out here http://rubies.travis-ci.org for Ubuntu 12.04 the 2.3.0 should be there

:+1:

**Comment by jkremser** [comment]
fixed by #62
